minidump/src/minidump.rs

```diff
@@ -1426,9 +1417,22 @@ impl MinidumpModuleList {
 
     /// Return a `MinidumpModule` whose address range covers `address`.
     pub fn module_at_address(&self, address: u64) -> Option<&MinidumpModule> {
-        self.modules_by_addr
-            .get(address)
-            .map(|&index| &self.modules[index])
+        let nearest_module = match self
+            .sorted_modules
+            .binary_search_by_key(&address, |i| self.modules[*i].base_address())
+        {
+            Ok(i) => &self.modules[self.sorted_modules[i]],
+            Err(0) => return None,
+            Err(i) => &self.modules[self.sorted_modules[i - 1]],
+        };
+        let end_address = nearest_module
+            .base_address()
+            .saturating_add(nearest_module.size());
```

Comment:
in theory this one should be validated on first digest so it's fine for it to panic..?